Website/README.md

```diff
@@ -0,0 +1,9 @@
+# Website
+**A README file can be described as documentation with guidelines on how to use a project. 
+Usually it will have instructions on how to download, install, run and contribute to the project but we've already worked on it which is the main [README](https://github.com/SaptarshiSarkar12/Drifty/blob/master/README.md) file of our project.**

```

Comment:
```suggestion
Usually it will have instructions on how to download, install, run and contribute to the project but we've already worked on it which is the main [README](https://github.com/SaptarshiSarkar12/Drifty/blob/master/README.md) file of our [project](https://github.com/SaptarshiSarkar12/Drifty).**

```